runtime/codert_vm/cnathelp.cpp

```diff
@@ -69,7 +69,7 @@ buildJITResolveFrameWithPC(J9VMThread *currentThread, UDATA flags, UDATA parmCou
 	currentThread->jitException = NULL;
 	resolveFrame->specialFrameFlags = flags;
 #if defined(J9SW_JIT_HELPERS_PASS_PARAMETERS_ON_STACK)
-	resolveFrame->parmCount = parmCount;
+	resolveFrame->parmCount = 0;
```

Comment:
Is this change intentional? If so then the `#if` and `#else` paths are identical and can be merged.